requirements.txt

```diff
@@ -16,7 +16,7 @@ bandit==1.6.1
 boto==2.49.0
 boto3==1.9.176
 botocore==1.12.176
-boxsdk==2.0.0a11
+boxsdk==2.6.0
```

Comment:
you only need this pinned here - not above. but please update to `boxsdk[jwt]==2.6.0`